server/settings.py

```diff
@@ -147,6 +151,9 @@
     "slugline": {
         "required": False,
     },
+    "categories": {
+        "required": True,
+    },
```

Comment:
I think this is preventing the upload atm, it's probably not working with custom cvs